contracts/ERC4626RateProvider.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.0;
+
+import "./interfaces/IRateProvider.sol";
+import "@openzeppelin/contracts/interfaces/IERC4626.sol";
```

Comment:
```suggestion
import "@openzeppelin/contracts/interfaces/IERC4626.sol";

import "./interfaces/IRateProvider.sol";
```
Import external dependencies first, then internal dependencies with spacing between to maintain repo convention.